src/manager.rs

```diff
@@ -519,6 +520,8 @@ where
         Ok(())
     }
 
+    /// Request that the primary device to encrypt & send all of its contacts as a message to ourselves
+    /// which can be then received, decrypted and stored in the message receiving loop
```

Comment:
I guess you could add a small comment about a post condition, that after this method is successfully done, the contacts are stored in the store.